go/pools/resource_pool.go

```diff
@@ -106,7 +71,6 @@ type (
 		idleTimer *timer.Timer
 		logWait   func(time.Time)
 
-		settingResources  chan resourceWrapper
 		getCount          atomic.Int64
 		getSettingCount   atomic.Int64
```

Comment:
settings-related counters can be removed.

Reply:
Fixed!